src/widgets/rooms.rs

```diff
@@ -24,6 +24,8 @@ use tokio::sync::RwLock;
 
 use crate::widgets::RenderWidget;
 
+use sublime_fuzzy::{best_match, format_simple};
```

Comment:
If you could add this `#[allow(unused_imports)]` and maybe a TODO. Or if you're planning on continuing this PR ignore me.

Reply:
I've added a TODO for now. Didn't have time yet to look at your latest comment